css/main.css

```diff
@@ -0,0 +1,97 @@
+@import url("https://fonts.googleapis.com/css2?family=DM+Sans:wght@400;500;700&display=swap");
```

Comment:
You can move this link to the typography section maybe

Reply:
Yes. I will. Actually this landing page is different from the component section. Later on I'll define typography at one place & then use it in all the components.